.teamcity/tests/buildTypes/E2ETests.kt

```diff
@@ -1,32 +1,41 @@
 package tests.buildTypes
 
+import jetbrains.buildServer.configs.kotlin.*
 import jetbrains.buildServer.configs.kotlin.BuildType
 import jetbrains.buildServer.configs.kotlin.buildFeatures.PullRequests
-import jetbrains.buildServer.configs.kotlin.buildFeatures.commitStatusPublisher
 import jetbrains.buildServer.configs.kotlin.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.buildSteps.dockerCompose
 import jetbrains.buildServer.configs.kotlin.buildSteps.script
-import jetbrains.buildServer.configs.kotlin.triggers.vcs
 
 
 object E2ETests : BuildType({
   name = "E2E tests"
 
   vcs {
     root(vcsRoots.KotlinLangOrg)
+  git("Kotlin_KotlinSites_KotlinlangTeamcityDsl_BuildReferenceDocs") {
+      name = "Kotlin_KotlinSites_KotlinlangTeamcityDsl_BuildReferenceDocs"
+      url = "git@github.com:JetBrains/kotlin-web-site.git"
+      branchSpec = """
+        +:refs/heads/(add-testing-page)
+    """.trimIndent()
+  }
   }
 
-  steps {
-    script {
-      scriptContent = "./scripts/test/up.sh"
-    }
-
-    script {
-      scriptContent = "./scripts/test/run.sh"
+    dependencies {
+        artifacts(AbsoluteId("Kotlin_KotlinSites_KotlinlangTeamcityDsl_BuildReferenceDocs")) {
```

Comment:
I believe the dependency can be resolved differently. You can import the class and use it (we do it already in the configuration).

Reply:
✅